File: level76/homework/codwars11.js
//https://www.codewars.com/kata/5282b48bb70058e4c4000fa7/train/javascript
function hexStringToRGB(h) {
  return {
    r: parseInt(h.slice(1,3), 16),
    g: parseInt(h.slice(3,5), 16),
    b: parseInt(h.slice(5,7), 16)
  };  
}

File: level 076/homework/codwars11.py
#https://www.codewars.com/kata/5282b48bb70058e4c4000fa7/train/python (RGB 2)
def hexToInt(hex_string):
    return int(hex_string, 16)

def hex_string_to_RGB(hex_string): 
    return {
        'r': hexToInt(hex_string[1:3]),
        'g': hexToInt(hex_string[3:5]),
        'b': hexToInt(hex_string[5:])
    }